gadget/gadget.go

```diff
@@ -454,14 +531,14 @@ func validateCrossVolumeStructure(structures []LaidOutStructure, knownStructures
 	return nil
 }
 
-func validateVolumeStructure(vs *VolumeStructure, vol *Volume) error {
+func validateVolumeStructure(vs *VolumeStructure, vol *Volume, constraints *ModelConstraints) error {
```

Comment:
No need to pass the constraints to `validateVolumeStructure` anymore

Reply:
Removed unnecessary parameter.